wordpress-mobile/WordPress-Android
# Plans in Site Creation: Update RELEASE-NOTES.txt

Fixes #19376

To test:

- `Launch` JP app
- `Go` to Me-> Debug settings
- `Switch` to `Remote Feature` tab, find `plans_in_site_creation` and enable it
-  `Switch` to **My Site** tab
- `Tap` the down arrow to the right of the site name on the My Site screen
- `Tap` the + icon at the top right on the Site picker screen
- `Select` ⊕ **Create WordPress.com site** on the modal dialog
- `Tap` Skip at top right on the first step (topic screen)
- `Tap` Skip at top right on the next step (theme screen)
- `Enter` a search query (whatever you want here)
- `Verify` that result domains are as shown in the screenshot above
- `Select` a free domain
- `Verify` Plans screen opens with Personal, Premium, Business, and Ecommerce plans
- `Select` a paid plan 
- `Verify` it opens checkout, after site setup when an annual paid plan is chosen

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
